layouts/shortcodes/observability_pipelines/configure_existing_pipelines/destination_env_vars/microsoft_sentinel.md

```diff
@@ -0,0 +1,4 @@
+- Data collection endpoint (DCE)
+	- Stored as the environment variable: `DD_OP_DESTINATION_MICROSOFT_SENTINEL_DCE_URI`
```

Comment:
```suggestion
	- Stored as the environment variable `DD_OP_DESTINATION_MICROSOFT_SENTINEL_DCE_URI`
```
If I'm introducing inconsistency, feel free to ignore

Reply:
Seems like I have both versions in the docs....I'll open a separate PR later to make it consistent.